examples/llm/__init__.py

```diff
@@ -0,0 +1,19 @@
+"""PyTorch Geometric LLM examples.
+
+Graph-based retrieval and conversation utilities for PyG.
+"""
+
+# WHG-Retriever components
+try:
+    from .whg_model import WarehouseGRetriever, WarehouseTaskHead  # noqa: F401
```

Comment:
again, examples should be standalone scripts, not modules. please move this stuff under the torch_geometric folder. Maybe these can go under torch_geometric/utils/data_warehouse.py.

(also open to suggestions on the naming of the utils file)